user_modules/shared_testcases/commonSteps.lua

```diff
@@ -457,6 +457,11 @@ function commonSteps:DeletePolicyTable()
 		print( " \27[33m commonSteps:DeletePolicyTable : policy.sqlite is not found \27[0m " )
 	end
 
+	local hmiCapsFile = config.pathToSDL .. SDLConfig:GetValue("AppStorageFolder") .. "/" .. SDLConfig:GetValue("HMICapabilitiesCacheFile")
```

Comment:
@iCollin Since `commonSteps` module and `DeletePolicyTable` mostly used in old script the main idea was to make new feature (Persisting hmi capabilities) 'invisible' for them. The easiest way to do this was to inject deleting of HMI capabilities cache file into some function which is used in preconditions.
Assumption was that this cache file can not exist without policy database. So if policy DB is removed it means cache file has to be removed  as well.